File: Vigilantes.DaprWorkshop.OrderService/java/order-service/src/main/java/vigilantes/daprworkshop/orderservice/controller/MenuItemController.java
package vigilantes.daprworkshop.orderservice.controller;

import java.util.List;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import vigilantes.daprworkshop.orderservice.model.MenuItem;

@RestController
public class MenuItemController {

    @GetMapping("/menuitem")
    public List<MenuItem> get(){    
        return MenuItem.getAll();
    }
    
}

File: Vigilantes.DaprWorkshop.OrderService/csharp/Controllers/MenuItemController.cs
using System.Collections.Generic;
using Microsoft.AspNetCore.Cors;
using Microsoft.AspNetCore.Mvc;
using Microsoft.Extensions.Logging;
using Vigilantes.DaprWorkshop.OrderService.Models;

namespace Vigilantes.DaprWorkshop.OrderService.Controllers
{
    [ApiController]
    [EnableCors]
    [Route("[controller]")]
    public class MenuItemController : ControllerBase
    {
        private readonly ILogger<MenuItemController> _logger;

        public MenuItemController(ILogger<MenuItemController> logger)
        {
            _logger = logger;
        }

        [HttpGet]
        public List<MenuItem> Get()
        {
            return MenuItem.GetAll();

        }
    }
}